.github/workflows/cd.yml

```diff
@@ -22,6 +22,26 @@ jobs:
     secrets:
       PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
 
+  standalone-executable:
+    name: Create a standalone linux executable
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@v1
+
+      - name: Build standalone binary
+        run: poetry run -- nox -s build-standalone-binary -- --executable-name "itde_linux_x86-64"
+
+      - name: Upload to release
+        run: gh release upload ${{ github.ref_name }} dist/itde_linux_x86-64
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Move into separate yml file and call workflow here, similar to cd-job above:
```
uses: ./.github/workflows/build-executable.yml
```

build-executable.yml then has an input parameter for the release which can be empty.
=> Call build-executable.yml also from CI with empty release.